packages/firestore/src/local/indexeddb_mutation_queue.ts

```diff
@@ -517,7 +515,7 @@ export class IndexedDbMutationQueue implements MutationQueue {
     let containsKey = false;
     return documentMutationsStore(txn)
       .iterate({ range: startRange, keysOnly: true }, (key, _, control) => {
-        const [userID, keyPath, batchID] = key;
+        const [userID, keyPath] = key;
```

Comment:
I don't love this change. Perhaps we should use the ignore-pattern option of  no-unused-variable to exclude _blah variables, and then name this _batchId or something?